guide/interactions/slash-commands.md

```diff
@@ -0,0 +1,1179 @@
+
+- Guild ID ✅
+- Command ID ✅
+- User/role ID ✅
+
+Finally, we make the HTTP PUT request in line 38. However, we use a different endpoint (route) this time: `Routes.guildApplicationCommandsPermissions()` (line 39), and we pass our array in the body of the PUT request (line 40).
+
+Congratulations! 🎉
+
+Go ahead and assign yourself that role in your guild. You should be able to use the `ping` command now. If you remove that role from yourself, you will see the `ping` command as disabled again.
+
+::: warning
+As you've probably noticed, we can only adjust the permissions of a command at the top-level. This means that we do not have the granularity of adjusting permissions for subcommand groups and/or subcommands.
+
+Let's use the `/permissions` command from [Subcommands and Subcommand groups](#subcommands-and-subcommand-groups) as example. It's NOT possible to enable `/permissions user get` for a user, while also disabling `/permissions user edit` for that same user.
+:::
+
+### Alternative
+
+You can also make permission overwrite with the bot itself.
+
+```js:no-line-numbers {5}
+const pingCommand = client.guilds.cache.get(guildId).commands.cache.find(cmd => cmd.name === 'ping');
+
+const permissions: [
+  {
+    id: '1234567890',
+    type: 'ROLE',
+    permission: true,
+  }
+];
+
+await pingCommand.permissions.set({ permissions });
+```
+
+All 3 IDs are satisfied here as well:
+
+- Guild ID ✅
+- Command ID ✅
+- User/role ID ✅
+
+The Guild ID is specified in `guildId`.
+
+The Command ID is built into the `pingCommand` variable, because it's an instance of [ApplicationCommand](https://discord.js.org/#/docs/main/stable/class/ApplicationCommand).
+
+The user/role ID is specified in the highlighted line.
```

Comment:
As mentioned earlier these topics where separate for a good reason, mashing it all together clutters the page and makes it quite unreadable, there's also a lot of redundant and out of place information to be found within this. It should be kept as it was.

Reply:
👍🏻 i will split my "single page" into three files. we can address the redundant and out of place information once ive done the split.